blog-content/articles/testing-makes-you-faster-day-one.md

```diff
@@ -10,7 +10,12 @@ summary: How testing makes you fast day one and not in some unknown future
 *"I am not writing tests right now as they are going to slow me down"* is one of the most common excuses I heard in 
 the projects I worked on.
 
-A lack of test writing experience can make you feel this way, but I think there is another, bigger reason for the perception that tests slow you down. I believe this reason is actually an untapped superpower: the power of exercising any piece of your software in `O(1)` keystrokes. 
+A lack of test writing experience can make you feel this way, but I think there is another, 
+bigger reason for the perception that tests slow you down. I believe this reason is actually an untapped superpower:
+the power of exercising any piece of your software in `O(1)` keystrokes. By `O(1)` I mean that regardless of how big 
+your application becomes you will always be one button away of seeing the impact of your changes.  
+
+The superpower of having the tightest feedback loop. 
```

Comment:
This is fantastic! Amazing work. 

I have one *tiny* change to ask for here.

Your summary is a claim that has two parts: The main reason you believe tests slow you down is something that is actually a huge well of unrealized potential
Part 1: This idea is one of the main reasons you may believe tests slow you down
Part 2: The idea is an unrealized superpower

You've done a fantastic job of making Part 2 clear, but Part 1 is still not clear. Why is this a main reason that people believe tests slow you down? I strongly recommend you put one line into this intro that clarifies this. Once you do so, the thesis will become crystal clear!

Reply:
This makes sense. Could be a good content for a next post.